app/models/operatingsystem.rb

```diff
@@ -46,6 +46,12 @@ class Operatingsystem < ApplicationRecord
 
   default_scope -> { order(:title) }
 
+  scope :sort_by_version, lambda {
+    to_a.sort_by do |os|
```

Comment:
I don't think `to_a` is needed since collections implement `Enumerable`
```suggestion
    sort_by do |os|
```